### ISSUE TITLE
update-notifier not exiting

### ISSUE THREAD

**Issue by vjpr** [pnpm/pnpm#1122]
```
ps aux | grep node
root      2155  0.0  1.3 1122104 26764 ?       Ssl  04:15   0:00 /usr/local/bin/node /usr/local/pnpm-global/1/node_modules/.registry.npmjs.org/pnpm/1.39.0/node_modules/pnpm/lib/node_modules/update-notifier/check.js {"pkg":{"name":"pnpm","version":"1.39.0"}}
vagrant   2182  0.0  1.2 1056576 26496 ?       Ssl  04:16   0:00 /usr/local/bin/node /usr/local/pnpm-global/1/node_modules/.registry.npmjs.org/pnpm/1.39.0/node_modules/pnpm/lib/node_modules/update-notifier/check.js {"pkg":{"name":"pnpm","version":"1.39.0"}}
root      2209  0.0  1.3 1056564 28520 ?       Ssl  04:17   0:00 /usr/local/bin/node /usr/local/pnpm-global/1/node_modules/.registry.npmjs.org/pnpm/1.39.0/node_modules/pnpm/lib/node_modules/update-notifier/check.js {"pkg":{"name":"pnpm","version":"1.39.0"}}
root      2263  0.0  1.3 1056560 28516 ?       Ssl  04:17   0:00 /usr/local/bin/node /usr/local/pnpm-global/1/node_modules/.registry.npmjs.org/pnpm/1.39.0/node_modules/pnpm/lib/node_modules/update-notifier/check.js {"pkg":{"name":"pnpm","version":"1.39.0"}}
root      2292  0.0  1.3 1056576 27536 ?       Ssl  04:18   0:00 /usr/local/bin/node /usr/local/pnpm-global/1/node_modules/.registry.npmjs.org/pnpm/1.39.0/node_modules/pnpm/lib/node_modules/update-notifier/check.js {"pkg":{"name":"pnpm","version":"1.39.0"}}
root      2493  0.1  1.2 991032 26532 ?        Ssl  04:19   0:00 /usr/local/bin/node /usr/local/pnpm-global/1/node_modules/.registry.npmjs.org/pnpm/1.39.0/node_modules/pnpm/lib/node_modules/update-notifier/check.js {"pkg":{"name":"pnpm","version":"1.39.0"}}
vagrant   3289  0.0  0.0  11740   932 pts/0    R+   04:20   0:00 grep --color=auto node
```

Maybe the updater is not using npm config proxy setting - because I am behind a corporate firewall.

### Additional information:

 - `node -v` prints: 8.1.2
 - Windows, OS X, or Linux?: Ubuntu 14

**Comment by vjpr** [comment]
Is there a way to disable the update checker?

**Comment by zkochan** [comment]
closed in favor of #2752
